src/Components/Input/Input.tsx

```diff
@@ -1,29 +1,23 @@
 import React from 'react';
 import StyledInput from './StyledInput';
-import { IInputField } from '../../types/input';
+import { IInputField, IOtherProps } from '../../types/input';
 
 const eventHandler = (e: any, removeClass: string, addClass: string) => {
   const inputWrapper = e.target.parentElement;
   inputWrapper.classList.remove(removeClass);
   inputWrapper.classList.add(addClass);
 }
 
-declare module 'react' {
-  interface HTMLAttributes<T> extends AriaAttributes, DOMAttributes<T> {
-    // extends React's HTMLAttributes
-    required?: boolean
-  }
-}
-
-const InputField = ({ id, required, placeholder, labelText, error }: IInputField) => (
-  <StyledInput required={ required }>
-    <label htmlFor={ id }>{ error ? "Error" : labelText }</label>
+const InputField = ({ label, field, form, ...props }: IInputField & IOtherProps ) => (
```

Comment:
Looks like this field will be used only in `formik ` forms, pelase rename to `FormikInputField`.
Also, please create folder `FormikFormControls` in `components` and relocate the component.